### DISCUSSION TITLE
Cool UI

### DISCUSSION THREAD

**Discussion by masterrite** [d2phap/ImageGlass#103] [General] [answered]
Cool UI, very clean.

But if you really want me or anyone else to use your program, don't meddle this with politics. You know what I mean. I'm not here to argue with you to whom somewhere really belongs to (though apparently our ideas clash), but I want to adamantly tell you please remove the default image that shows up upon startup, or change it. Because of that I've deleted your program: the pleasure and delight at the clean UI and functions ImageGlass offers are eliminated by that default photo that showed up. I'm sorry but there are other image viewers out there. Yours could've stood out, though frankly I think it already did.

To recap, bad user experience. Though it's not tech-related, it still matters.

**Comment by Baegus** [comment]
+1

**Comment by d2phap** [comment, marked as answer]
Hi @masterrite 
Thanks for the feedback!
